lib/helpers/ApiHelpers.ts

```diff
@@ -64,10 +67,22 @@ export class ApiHelpers {
     return 'Bearer ' + someObject.access_token;
   }
 
+  async getCookie() {
+    let someStorage = await this.page.context().storageState();
+    
+    let cookieString = "";
+    
+    for(let cookie of someStorage.cookies){
+      cookieString += cookie.name + '=' + cookie.value+ ';';
+    }
+    
+    return cookieString;
+  }
   async get(url: string, params?: { [key: string]: string | number | boolean; }) {
     const options = {
       headers: {
-        'Authorization': await this.getBearerToken(),
+        'Authorization' : await this.getBearerToken(),
+        'Cookie': await this.getCookie(),
```

Comment:
Seems like we're duplicating this for all request (post/delete etc.)
maybe we should have a `getHeaders` method instead if they're the same for all ? 🧐